client/components/seo/preview-upgrade-nudge/index.jsx

```diff
@@ -73,32 +72,26 @@ export const SeoPreviewNudge = ( {
 								className="preview-upgrade-nudge__features-list-item-checkmark"
 								icon="checkmark"
 							/>
-							{ preventWidows(
-								translate(
-									"Preview your site's content as it will appear on Facebook, Twitter, and the WordPress.com Reader."
-								)
+							{ translate(
+								"Preview your site's content as it will appear on Facebook, Twitter, and the WordPress.com Reader."
 							) }
 						</li>
 						<li className="preview-upgrade-nudge__features-list-item">
 							<Gridicon
 								className="preview-upgrade-nudge__features-list-item-checkmark"
 								icon="checkmark"
 							/>
-							{ preventWidows(
-								translate(
-									'Control how page titles will appear on Google search results and social networks.'
-								)
+							{ translate(
+								'Control how page titles will appear on Google search results and social networks.'
 							) }
 						</li>
 						<li className="preview-upgrade-nudge__features-list-item">
 							<Gridicon
 								className="preview-upgrade-nudge__features-list-item-checkmark"
 								icon="checkmark"
 							/>
-							{ preventWidows(
-								translate(
-									'Customize your front page meta data to change how your site appears to search engines.'
-								)
```

Comment:
I'm not entirely sure if this nudge is still in use; site previews on free or personal plans don't seem to display this type of upgrade prompt. In any case the affected text is inside a list so list styles should apply.

Reply:
From memory I _think_ the preview where this might have appeared could have occurred on the Calypso version of the pages screen (which I don't think is used anymore), or the Calypso-ified iframed Gutenberg, which also isn't used any more. In any case, I wasn't able to find any obvious uses of it, and your logic here sounds fine to me!